contracts/libraries/State.sol

```diff
@@ -101,9 +112,10 @@ struct VaultState {
     VaultPool pool;
     uint256 initialDsPrice;
     // will be set to true after first deposit to LV. 
-    // to prevent manipulative behavior when depositing to Lv since we depend on preview redeem eearly to get 
+    // to prevent manipulative behavior when depositing to Lv since we depend on preview redeem early to get 
     // the correct exchange rate of LV
     bool initialized;
+    mapping(address => LvInternalBalance) userLvBalance;
```

Comment:
This won't work and will corrupt the storage, since all the state live in the same slot this would break the current struct encoding. for now, we can store this info in the `moduleState.sol` under the state mapping, this would give it a unique storage slot